docs/install/install-cave-beta-v20.md

```diff
@@ -163,3 +234,6 @@ sudo yum remove awips2-*
 rm -rf /awips2/cave
 rm -rf ~/caveData
 ```
+
+
+[def]: ../images/E
```

Comment:
I think this very last line might have come from my commit? but I don't know what it is or if it needs to be here at all?

Reply:
Okay, I've got it removed now